pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/invertedindex/InvertedIndexHandler.java

```diff
@@ -68,14 +68,14 @@ public InvertedIndexHandler(@Nonnull File indexDir, @Nonnull SegmentMetadataImpl
   }
 
   public void createInvertedIndices()
-      throws IOException {
+      throws Exception {
```

Comment:
Why changing this?

Reply:
Sorry, this was a temp change. I forgot to cleanup before creating PR. 
Removed.